Tasks/types.js

```diff
@@ -1,18 +1,18 @@
 // Count types in an array
 
-types_ = function (s) {
-  types_ = {
+'use strict';
+
+const countTypes = (array) => {
```

Comment:
```suggestion
const countTypes = array => {
```
Here you can also do it without brackets (  ), but it`s your choice.